llama_stack/cli/stack/_build.py

```diff
@@ -371,14 +374,19 @@ def _run_stack_build_command_from_build_config(
         if not image_name:
             raise ValueError("Please specify an image name when building a venv image")
 
+    # At this point, image_name should not be None due to the validation above
```

Comment:
why add this?

Reply:
I thought it might be useful to validate whether image_type is one of the supported options (conda, venv, or container).